packages/lit-dev-content/site/css/blog-post.css

```diff
@@ -1,140 +1,85 @@
 @import url('./blog-common.css');
```

Comment:
You might want to revert this file as it [breaks blogposts.](https://pr909-202d83b---lit-dev-5ftespv5na-uc.a.run.app/blog/2022-02-07-eleventy/) Any changes you need to styles in articles should live in `articles.css`

Reply:
Weird. These aren't actually my changes, I must have introduced something funky when I was rebasing.